src/properties.cpp

```diff
@@ -5033,8 +5033,8 @@ const XmpPropertyInfo* XmpProperties::propertyInfo(const XmpKey& key) {
   std::string property = key.tagName();
   // If property is a path for a nested property, determines the innermost element
   if (auto i = property.find_last_of('/'); i != std::string::npos) {
-    for (; i != std::string::npos && !isalpha(property.at(i)); ++i) {
-    }
+    while (!std::isalpha(property.at(i)))
```

Comment:
This should probably check for `\0`. I agree that the previous `i != std::string::npos` check was wrong.